mailpoet/lib/Cron/Daemon.php

```diff
@@ -61,6 +62,8 @@ public function run($settingsDaemonData) {
           $worker->process($this->timer); // BC for workers not implementing CronWorkerInterface
         }
       } catch (\Exception $e) {
+        Helpers::mySqlGoneAwayExceptionHandler($e);
```

Comment:
Just want to make sure I understand. Here, we intentionally ignore the returned message and call the helper just to log the error via error_log. Correct?

Reply:
Yup, that's correct.